### PR TITLE
Time machine

### PR DESCRIPTION
**Pull request by cmoesgaard**
Added some tests and a fix for an edge case where we attempt to get an object outside of the validity period of its properties, but where metadata still exists at that given time.

### CODE REVIEW

File: mora/service/common.py [hunk 1]
```diff
@@ -459,3 +459,15 @@ def get_valid_to(obj, fallback=None):
...
+def is_reg_valid(reg, gyldighed_key):
+    """
+    Check if a given registration is valid
+
+    :param reg: A registration object
+    :param gyldighed_key: A key for where to find the gyldighed on the object
+    :return: True if registration 'gyldighed' is 'Aktiv'
+    """
+    return bool(get_obj_value(reg, ('tilstande', gyldighed_key),
+                              lambda x: x.get('gyldighed') == 'Aktiv'))
```
Comment (danchr) [inline mora/service/common.py]: Couldn't you simplify this code a bit by iterating over the states, and check if any are active?

Comment (cmoesgaard) [reply mora/service/common.py]: That is essentially what's happening already. But it could probably be simplified a bit.

Comment (danchr) [reply mora/service/common.py]: I mean that you could iterate over all states rather than specify `gyldighed_key` — that seems a bit cleaner to me?

Comment (cmoesgaard) [reply mora/service/common.py]: That's a good point
